languages/csharp/exercises/concept/resource-lifetime/.docs/instructions.md

```diff
@@ -0,0 +1,42 @@
+You are back working on the ORM (Object Relationship Mapping) system introduced in (TODO cross-reference-tba).
+
+Our ORM usage analysis shows that 95% of transactions are executed from within one calling method, and it has been decided that it would be more appropriate to have a single ORM method that opened, wrote and committed a transaction.
+
+The database has the following instance methods:
+
+- `Database.BeginTransaction()` starts a transaction on the database.
+- `Database.Write(string data)` writes data to the database within the transaction. If it receives bad data an exception will be thrown. An attempt to call this method without `BeginTransction()` having been called will cause an exception to be thrown. If successful the internal state of the database will change to `DataWritten`.
+- `Database.Commit()` commits the transaction to the database. It may throw an exception if it can't close the transaction of if `Database.BeginTransaction()` had not been called.
```

Comment:
```suggestion
- `Database.Commit()` commits the transaction to the database. It may throw an exception if it can't close the transaction or if `Database.BeginTransaction()` had not been called.
```